lib/command.js

```diff
@@ -28,4 +32,16 @@ function getClientClass (protocol = 'ftp') {
   }
 }
 
+const validExtensions = ['.pem', '.key', '.pub']
```

Comment:
Can you also make it possible to mount a file without any extension, since the standard name of generated private key file is `id_ed25519`, `id_rsa` and so on?

Reply:
I see, I'll do the changes